### DISCUSSION TITLE
Welcome to OpenScan-PCB Discussions!

### DISCUSSION THREAD

**Discussion by OpenScanEu** [OpenScanEu/OpenScan-PCB#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WhatNoAlan** [comment]
Completely new to the GitHub world. I am retired and enjoy 3D printing. My background is research and development of Avionics systems hardware. Hoping to get some good results with the OpenScan design. Tried others and not pleased. I primarily will use the device for scanning in small 1/64 diecast cars for printing in resin. Even at 74 years old I still play with slot cars! :)

**Reply by OpenScanEu** [reply]
This is amazing and I am so grateful that people with very different backgrounds, age and nationality join this project to create something new :) thank you so much for taking the time to share your experience and ideas!

**Comment by stefan63** [comment]
I just love this project but "Just a thought" wouldn't it be nice to be able to use a cheap 3D printer motherboard to run the firmware on?  Kind regards/Stefan

**Reply by probably-Erwins-Cat** [reply]
A 3D-printing board would work nicely for controlling the steppers and such, however, a simple board does not have the functionality of connecting a camera and writing with adequate speed on an SD card etc. 
Advanced printer firmware for printer, eg klipper, are using for that reason a rpi as parallel processsing unit, like openscan is doing. Further, you need various voltage levels which is typically solved by using dedicated power source(s). 

While a cheap 3D printing board can make certain aspects easier, you then run into other difficulties.

**Comment by timhaller** [comment]
Hello 👋,
I just have a quick question, are the files for the PCB available somewhere? I can't seem to find them...

**Reply by probably-Erwins-Cat** [reply]
The schematics are provided on the website, or here:
https://drive.google.com/file/d/15dHPhmN0gLu5s5aGFLwUulbkThmojQXg/view
